src/elastic-search.js

```diff
@@ -27,22 +27,38 @@ module.exports = function sendTestResults(testResultsLog, done) {
     var uuidRegex = /\[(uuid:.*?)\]/g;
     var timestamp = new Date().getTime();
     for (var arrayLength = 0; arrayLength < passedTestDataSize; arrayLength++ ) {
+
+      let contextValues; 
+      if(passedTestData[arrayLength].context !== undefined){
```

Comment:
code formatting doesn't match... on line 29 there's a space between for and begin paren, also a space between end paren and bracket... should keep this consistent throughout the code

Reply:
fixed in next checkin